test/TestProjects/MediaTypes-TypeSpec/src/Generated/tspCodeModel.json

```diff
@@ -116,38 +116,7 @@
        "Name": "contentType",
        "NameInRequest": "content-type",
        "Type": {
-        "$id": "17",
-        "Kind": "Union",
-        "Name": "Union",
-        "UnionItemTypes": [
-         {
-          "$id": "18",
-          "Kind": "Literal",
-          "Name": "Literal",
-          "LiteralValueType": {
-           "$id": "19",
-           "Kind": "Primitive",
-           "Name": "String",
-           "IsNullable": false
-          },
-          "Value": "application/json; serialization=Avro",
-          "IsNullable": false
-         },
-         {
-          "$id": "20",
-          "Kind": "Literal",
-          "Name": "Literal",
-          "LiteralValueType": {
-           "$id": "21",
-           "Kind": "Primitive",
-           "Name": "String",
-           "IsNullable": false
-          },
-          "Value": "application/octet-stream",
-          "IsNullable": false
-         }
-        ],
-        "IsNullable": false
+        "$ref": "2"
```

Comment:
"union to enum" is handled by TCGC now, therefore this changed from a union, to a enum (id 2 refers a enum)